src/components/Table/Table.qa.stories.tsx

```diff
@@ -378,3 +388,188 @@ const flatRows: GridDataRow<BeamRow>[] = [
     },
   })),
 ];
+
+/* Input Field Story and data */
+type InputFieldRows = SimpleHeaderAndData<AuthorInput>;
+
+export function InputFieldStates() {
+  const { getFormState } = useFormStates<AuthorInput>({
+    config: formConfig,
+    getId: (o) => o.id!,
+  });
+
+  return (
+    <>
+      <h2 css={Css.mb1.$}>Fixed Table Styles</h2>
+      <GridTable<InputFieldRows>
+        style={beamFixedStyle}
+        columns={inputFieldColumns(getFormState)}
+        rows={inputFieldRows}
+        stickyHeader
+      />
+
+      <h2 css={Css.mt5.mb1.$}>Flexible Table Styles</h2>
+      <GridTable<InputFieldRows>
+        style={beamFlexibleStyle}
+        columns={inputFieldColumns(getFormState, true)}
+        rows={inputFieldRows}
+        stickyHeader
+      />
+    </>
+  );
+}
+
+function inputFieldColumns(getFormState: (author: AuthorInput) => ObjectState<AuthorInput>, flexible?: boolean) {
+  function applyStateProps(id: string) {
+    return {
+      internalProps: { forceFocus: id === "a:3", forceHover: id === "a:2" },
+      errorMsg: id === "a:4" ? "Description of error" : undefined,
+    };
+  }
+  return [
+    column<InputFieldRows>({
+      header: "Name",
+      data: (data, row) => ({
+        content: () => {
+          const os = getFormState(data);
+          return <BoundTextField field={os.firstName} {...applyStateProps(row.id)} />;
+        },
+      }),
+    }),
+    column<InputFieldRows>({
+      header: "Biography",
+      data: (data, row) => ({
+        content: () => {
+          const os = getFormState(data);
```

Comment:
Wonder if we should eventually have `textAreaCell` type things that return the content + value `GridCellContents`... Nothing for now, just talking out loud.